src/ReviewService.Abstractions/ReviewService.Extensions.cs

```diff
@@ -1,7 +1,6 @@
 ﻿using System;
 using System.Threading;
 using System.Threading.Tasks;
-using ReviewService.Abstractions;
 
 namespace ReviewService;
```

Comment:
shouldn't that be in `ReviewService.Abstractions` namespace as well?

Reply:
We decided that this namespace shouldn't exist. I'm not sure how to explain it properly. @jeanplevesque

Reply:
In my opinion this is just because the abstraction package is just a way to make only the abstraction part available for each type of project but it should not mean that it has to have a different namespace. Usually our services have the same namespace for the interface and implementation.

Reply:
Exactly

Reply:
Oh I misread the changes. My mistake